modules/rules_build_error/0.6.0/presubmit.yml

```diff
@@ -0,0 +1,19 @@
+---
+bcr_test_module:
+  module_path: "examples"
+  matrix:
+    platform:
+      - "debian10"
+      - "ubuntu2004"
+      - "macos"
+      - "windows"
```

Comment:
```suggestion
```
I was attempting to support windows from 0.6.0, but for now let me just drop its support. I'll fix the issue of symlink in the next release 0.6.1